src/tab/Tab.tsx

```diff
@@ -13,32 +14,13 @@ export const Tab: React.VFC<TabProps> = ({
 }) => {
   return (
     <MuiTab
-      // css={[
-      //   css`
-      //     ${tw`text-primary-medium-default`}
-
-      //     &.MuiTab-root {
-      //       min-width: ${minWidth}px !important;
-      //     }
-
-      //     &.MuiTab-textColorPrimary {
-      //       ${tw`focus:outline-none!`}
-      //     }
-
-      //     &.Mui-selected {
-      //       ${tw`(focus:outline-none text-primary-dark-default)!`}
-
-      //       > .MuiTab-wrapper {
-      //         ${tw`text-primary-dark-default!`}
-      //       }
-      //     }
-
-      //     > .MuiTab-wrapper {
-      //       ${tw`text-shade-medium-default!`}
-      //     }
-      //   `,
-      //   twin,
-      // ]}
+      classes={{
+        // eslint-disable-next-line tailwindcss/no-custom-classname
+        root: clsx([`min-w-[${minWidth}] text-primary-medium-default`]),
```

Comment:
フォントサイズとboldとpaddingの設定をお願いします！

```suggestion
        root: clsx([`min-w-[${minWidth}] min-h-[auto] text-primary-medium-default text-r font-bold font-sans pt-2 pb-2.5`]),
```